util/src/main/scala/geotrellis/util/ResourceReader.scala

```diff
@@ -0,0 +1,10 @@
+package geotrellis.util
+
+import java.io._
+
+object ResourceReader {
+  def asString(filename: String): String = {
```

Comment:
This is for test functionality, not sure it's worth putting in the core utils JAR